### PR TITLE
Add system fields

### PR DESCRIPTION
**Pull request by Safintim**
Hi! I would like to participate in a project. I am using this library and have run into problems. I will make some pr, do you mind?

This pr for system fields. Every record has system fields, which are listed here https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/system_fields.htm

### CODE REVIEW

File: simple_mockforce/virtual.py [hunk 1]
```diff
@@ -91,7 +92,7 @@ def query(self, soql: str):
         order_by = (
             parse_results["order_by"].asList() if "order_by" in parse_results else None
         )
-        sobjects = self.get_sobjects(sobject)
+        sobjects = self.get_sobjects(sobject, include_deleted=True)
```
Comment (brno32) [inline simple_mockforce/virtual.py:95]: Shouldn't we only include deleted objects if the query explicitly asks for them? I believe that's SOQL's default behavior

**Review by brno32 — CHANGES_REQUESTED** [review]

**Comment by Safintim** [comment]
The test fell because of the use of different timing. I suggest introducing the freezegun library https://github.com/spulec/freezegun. Are you okay with refactoring?

**Comment by brno32** [comment]
Yes I'd be okay with using it! I've never used `freezegun`. Do you know how to configure it?

**Comment by Safintim** [comment]
Yes, it's not hard. I rewrote the test_crud_lifecycle using the `freezegun` library. 

I suggest in the future to work on tests and implement test factories (faker.Factory).

**Review by brno32 — APPROVED** [review]
Thank you so much for this contribution!

**Comment by brno32** [comment]
I've released these changes in `0.4.0`

**Pull request merged**